plugins/modules/docker_container_exec.py

```diff
@@ -183,6 +194,15 @@ def main():
     stdin = client.module.params['stdin']
     strip_empty_ends = client.module.params['strip_empty_ends']
     tty = client.module.params['tty']
+    env = client.module.params['env']
+
+    if env is not None:
+        for name, value in sorted(env.items()):
```

Comment:
not there's anything wrong with it, but just wondering why this is sorted?

Reply:
I think I wanted to write `list`, not `sorted`. The result is the same, except that `sorted` is a bit slower... Will change it.